### PR TITLE
Added a pull request template

### PR DESCRIPTION
**Pull request by AdityaAshvin**
A pull request template is added. It contains a Description section, Checklist Section and screenshot section.

Fixes: issue #182

### CODE REVIEW

**Comment by AdityaAshvin** [comment]
Please review it @SKAUL05 .

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Include a summary of the changes made and mention the issue it fixes. 
+
+Fixes # (include the issue number here)
+
+# Checklist:
+
+- [ ] My code follows the [guidelines](https://github.com/ankitdobhal/Awesome-Python-Scripts/blob/master/Contribution.md) of this project.
+- [ ] I have commented my code.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have added all the reviewers: @ankitdobhal and @SKAUL05
```
Comment (SKAUL05) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Remove this line not necessary. (Reviewers one )
Add:
-  I have followed all the appropriate naming conventions.

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
@ankitdobhal Have a look at this PR for any remaining changes.

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
LGTM 👍 
@ankitdobhal Review and Merge

**Comment by SKAUL05** [comment]
Duplicate already issue raised and participant working on it #97

**Pull request closed without merging**